python/ray/_private/runtime_env/packaging.py

```diff
@@ -799,18 +833,13 @@ def unzip_package(
         zip_ref.extractall(target_dir)
     if remove_top_level_directory:
         top_level_directory = get_top_level_dir_from_compressed_package(package_path)
-        if top_level_directory is None:
-            raise ValueError(
-                f"The zip package at {package_path} must contain "
-                "a single top level directory. Make sure there "
-                "are no hidden files at the same level as the "
-                "top level directory. You can ensure this by running "
-                "`zip -r example.zip example_dir` from the parent "
-                "directory of example_dir when creating the zip file. "
-                "You can check the contents with `zipinfo -1 example.zip`."
-            )
+        if top_level_directory is not None:
```

Comment:
This is always true right? because top_level_directory is a boolean from the args

Reply:
You might be thinking of `remove_top_level_directory`?

Reply:
`get_top_level_dir_from_compressed_package` returns `None` if there isn't a unique top-level directory (not counting `__MACOS`